packages/commons/src/logging/index.ts

```diff
@@ -45,6 +47,7 @@ const logFormat = (
   const correlationLogPart = correlationId
     ? `[CID=${correlationId}]`
     : undefined;
+  const spanIdLogPart = spanId ? `[SPNID=${spanId}]` : undefined;
```

Comment:
it is worth to add the last character 😄 
```suggestion
  const spanIdLogPart = spanId ? `[SPANID=${spanId}]` : undefined;
```